src/elasticsearch/elasticsearch-sync.service.ts

```
@@ -0,0 +1,766 @@
+      this.logger.debug(`Elasticsearch disabled, skipping application sync for user: ${userId}, cohort: ${cohortId}`);
+      return;
+    }
+
+    try {
+      this.logger.debug(`Starting application sync for user: ${userId}, cohort: ${cohortId}`);
+
+      // Get existing user document from Elasticsearch
+      const userDoc = await this.userElasticsearchService.getUser(userId);
+      
+      // Extract the application array if present
+      const source = userDoc && userDoc._source
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)